lib/keen-query/query-methods.js

```diff
@@ -58,6 +64,16 @@ const methods = {
 
 	interval: function (interval) {
 		const instance = this._getInstance();
+
+		// Keen extractions are different to all other queries. They do not support grouping or intervals.
+		if (this.extraction === "extraction") {
+			console.log("Removing instance() from the query as it's not supported in keen extractions.");
+			return instance;
+
+			// Todo: Decide whether it's better to "magically fix" the query, or throw an error
+			// throw new Error('Keen extractions are different to all other queries. They do not support grouping or intervals');
```

Comment:
if the keen API doesn't error my first preference would be just ignore and do nothing. Second to that error